### PR TITLE
qa: added basic-health-ok test script

### PR DESCRIPTION
**Pull request by smithfarm**
This has been tested in teuthology/OVH.

### CODE REVIEW

**Comment by smithfarm** [comment]
This has been tested in teuthology/OVH.

File: qa/README [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+The DeepSea integration suite residing in qa/ consists of a number of
+executable files (bash scripts), all of which make the following assumptions:
+
+1. there are at least two test nodes (VMs or physical machines) that are
+   running the same OS (e.g. Leap 42.3) and can see eachother over the network. 
+2. all the test nodes are configured as a Salt cluster, i.e.:
+   a. one node is configured as both a master and a minion
+   b. the remaining nodes are configured as minions only
+   c. the master can "salt '*' test.ping" all the minions
+3. the nodes have at least one external drive (>= 30GB) for OSD
```
Comment (jschmid1) [inline qa/README]: Does the drive really need to be that big?

Comment (jan--f) [inline qa/README]: worked for me with 10GB drives...so I guess no.

Comment (jschmid1) [inline qa/README]: great, no objections then! lgtm

**Pull request merged**